backend/dataall/api/Objects/Environment/resolvers.py

```diff
@@ -699,7 +699,51 @@ def get_pivot_role_template(context: Context, source, organizationUri=None):
             )
             raise e
 
+def get_cdk_exec_policy_template(context: Context, source, organizationUri=None):
```

Comment:
Instead of using the utils.Parameter class could you re-use the ParameterStore that is imported globally? This is to centralize the boto3 calls (Parameter will be removed in future releases)

Reply:
I just realized that you were following the example of the pivotRole template. So most of my comments here are not applicable for this PR, we will open a separate issue to improve how we get both the pivotRole template and the cdkExec role template in a separate PR.